### DISCUSSION TITLE
GSoC 2024 Announcements

### DISCUSSION THREAD

**Discussion by adrianboguszewski** [openvinotoolkit/openvino#21963] [Google Summer of Code]
This discussion is for Google Summer of Code 2024 announcements only. 

If you want to ask questions about GSoC, please create a new discussion in the "Google Summer of Code" category.

**Comment by adrianboguszewski** [comment]
I'm happy to announce that OpenVINO Toolkit has been accepted as a mentoring organization for GSoC 2024! Dear contributors, please consider joining our organization :) More info: https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code

**Comment by adrianboguszewski** [comment]
We received 51 applications! Thank you for that! Right now, we're starting evaluating them :)
Please note just one person can be a contributor to each project.

Number of proposals for projects:
1. OpenVINO Extension for Automatic1111 Stable Diffusion WebUI - 3
2. Inference of Neural model in Node.js environment using OpenVINO - 2
3. Desktop Chat-Bot application - 4
4. Add Image-to-Image and Image-to-Text generation with GUI support for OpenVINO GenAI - 5
5. OpenVINO CPU plugin functional test cases WASM enablement - 0
6. OpenVINO Conditional Compilation improvement - 1
7. OpenVINO adapters - PyTorch adapter - 3
8. Improve Generative AI workload performance on ARM devices with OpenVINO - 4
9. NNCF Quantization Analytics Tool - 2
10. Accelerating PyTorch Lightning and ComfyUI with torch.compile OpenVINO - 1
11. PyTorch Model Optimizations with torch.compile OpenVINO Backend - 2
12. Prototype for JAX/Flax Models Support - 3
13. A general OpenVINO performance benchmarking suite for Vision-LLM models - 4
14. Add time-series anomaly detection support to Anomalib - 6
15. AnomalyGPT: Integrating Vision Language Models (VLMs) in Anomalib for zero- and few-shot anomaly detection - 2
16. Synthetic Anomaly Generation for Threshold Selection and Model Evaluation in Anomalib - 0
17. Extending OpenVINO's llama.cpp community plugin for configurable GGML inferencing backend - 0
18. Benchmarking PyTorch Performance with TorchBench on EC2 Instances - 1
19. OpenVINO Messenger AI-Assistant for an AI PC - 3
20. Eyes health utilizing OpenVINO and Neural Processing Unit (NPU) on an AI PC – 4
21. Porting and Optimizing OpenVINO for RISC-V Devices with Vector Extension - 1

**Comment by adrianboguszewski** [comment]
Dear contributors,

A few days back, Google announced the projects for this summer. We asked for 17 projects but finally got 11 slots, and we will work with 11 contributors on 11 projects (1, 2, 4, 7, 8, 11, 12, 15, 19, 20, 21). The competition was quite high (we got 51 applications in total), so we had to decide what contributor we would like to continue the work with. So if you didn't get in, it doesn't mean your application was bad. Even if the second or third application is great, the winner takes all, as there can be just one contributor per project. Moreover, your project might be in these 6 slots, which Google didn't assign us, which is very frustrating :( But feel free to reach out to the mentors to get feedback anyway.

If you are still interested in open-source and OpenVINO, you can continue your work by contributing to our OpenVINO repositories. We also encourage you to join the [Intel Software Innovator Program for Edge](https://www.intel.com/content/www/us/en/developer/community/edge-innovator-program.html), where you can earn some rewards. Please visit the program page to learn more.

In conclusion, thank you for your engagement, PRs to the OpenVINO repository, and your submissions to GSoC. Hope to see you next year!

And congratulations to all accepted contributors :)
